src/app/auth/login/login.component.ts

```diff
@@ -21,8 +21,6 @@ export class LoginComponent implements OnInit {
     });
   }
 
-  showErrorMessage = false;
```

Comment:
since this line was removed are you instantiating it anywhere else or is this a built in feature of angular

Reply:
this was added previously by me and then this had conflict with your progress on this login component thats why i took my changes back.